examples/webhook.ts

```diff
@@ -0,0 +1,94 @@
+import { SpvWalletClient, WebhookManager, WebhookHttpHandler, RawEvent } from '../dist/typescript-npm-package.cjs.js';
+import { exampleAdminKey } from './example-keys.js';
+import { errMessage } from './utils.js';
+import Fastify, { FastifyReply, FastifyRequest } from 'fastify';
+
+// NOTE: ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+// THIS IS AN EXAMPLE ON HOW YOU CAN SET UP A WEBHOOK NOTIFIER
```

Comment:
If we can also update internal examples README.md for this file it would be awesome - separated paragraph with some use cases would be very useful 🙏